src/pages/query-view/query-view.js

```diff
@@ -86,8 +86,13 @@ const cleanseRow = row => {
 const cleanseField = value => {
   if (typeof value === "boolean") {
     return value.toString();
+  } else if (typeof value === "object") {
+    return JSON.stringify(value);
+  } else if (value === undefined) {
+    return ''
```

Comment:
Might want to check for `null` and `NaN` in addition to `undefined`.

Reply:
We learned that undefined is unnecessary but added null and NaN.